test/active_model_serializers/register_jsonapi_renderer_test_isolated.rb

```diff
@@ -111,18 +112,12 @@ def test_jsonapi_parser_registered
     end
 
     def test_jsonapi_renderer_registered
-      expected = {
-        'data' => {
-          'attributes' => {
-            'name' => 'Johnny Rico'
-          },
-          'type' => 'users'
-        }
-      }
+      expected = { 'name' => 'Johnny Rico' }
+
       payload = '{"data": {"attributes": {"name": "Johnny Rico"}, "type": "authors"}}'
       headers = { 'CONTENT_TYPE' => 'application/vnd.api+json' }
       post '/render_with_jsonapi_renderer', params: payload, headers: headers
-      assert expected, response.body
+      assert_equal expected, JSON.parse(response.body)['data']['attributes']
```

Comment:
Oh, this is because of the PORO `Author` having relationships that are explicit now, whereas before they were sort of squishy...

Fix is we should be explicit what our model and serializer are:

```ruby
class JsonApiRendererTest < ActionDispatch::IntegrationTest
  include ActiveSupport::Testing::Isolation
  class Author < ::Model
    attributes :name
  end
  class AuthorSerializer < ActiveModel::Serializer
    type: 'users'
    attribute :id
    attribute :name
  end
```

I think that's a better expectation, no?